types/theme.d.ts

```diff
@@ -163,72 +166,142 @@ export interface IThemePrepared {
 }
 
 export interface IThemeComponentStylesInput {
+  [key: string]: any
+
```

Comment:
As in the previous comment, can we make this: `[key: string]: IComponentPartStylesInput`? What else can be the value of this?

Reply:
Yes, here we can since all keys have the same value type.  Same with IThemeComponentStylesPrepared below which I have updated as well.